verifyCredentials.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable camelcase */
+const { PubSub } = require('@google-cloud/pubsub');
+const logger = require('@elastic.io/component-logger')();
```

Comment:
verify credentials function supports this.logger. Please replace it.

Reply:
replaced